plugins/out_kinesis_firehose/firehose.h

```diff
@@ -41,7 +41,7 @@ struct flush {
     size_t data_size;
 
     /* log records- each of these has a pointer to their message in tmp_buf */
-    struct event *events;
+    struct firehose_event *events;
```

Comment:
Same comments as merge to master review. Is this not a breaking change?

Reply:
Its just renaming an internal data structure so the name won't conflict with other structures in the code base i think, as long as it builds its good

Reply:
Sounds good